### PR TITLE
Added androidresource-checker-qual module

### PR DESCRIPTION
**Pull request by aagarwal1012**
@wmdietl, Please review this PR.

- I have created a module named `androidresource-checker-qual` which contains only the type qualifiers.
- It copies the content present in the `qual/` folder to the module and generates the `jar` file.
- Updated `.gitignore` file.

### CODE REVIEW

File: androidresource-checker-qual/build.gradle [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        attributes('Export-Package': '*')
+    }
+}
+
+compileJava {
+    dependsOn copySources
+}
+
+clean {
+    delete file('generated-src/')
```
Comment (wmdietl) [inline androidresource-checker-qual/build.gradle]: Above on line 27 it uses `into file('src/main/java')`. Shouldn't those two be consistent?
Can you test whether the `clean` target actually works correctly?
If this needs to be fixed, can you also make a PR against the Checker Framework sources.

**Review by wmdietl — COMMENTED** [review]
The manual should explain what the -qual.jar file is for.
Once you addressed the one comment I have, please make a similar PR to the template-for-a-checker repo.

File: androidresource-checker-qual/build.gradle (hunk 1 above)
Comment (aagarwal1012) [reply androidresource-checker-qual/build.gradle]: I have replaced the `generated-src` to `src`. I will make a PR to the CF sources.

Comment (aagarwal1012) [reply androidresource-checker-qual/build.gradle]: I have added the corresponding PR to the main CF repo, [link](https://github.com/typetools/checker-framework/pull/2688).

**Comment by aagarwal1012** [comment]
Made a similar PR in `templatefora-checker`, [link](https://github.com/typetools/templatefora-checker/pull/11).

**Pull request merged**